curriculum/challenges/english/14-responsive-web-design-22/build-a-personal-portfolio-webpage-project/build-a-personal-portfolio-webpage.md

```diff
@@ -280,3 +280,12 @@ p{
   height: 100vh;
 }
 ```
+
+---
+
+```html
+<head><style>@media (max-width: 500px){nav{display: none;}}</style></head><body><nav id="navbar"><a href="#projects">text</a> | </nav><main><section id="welcome-section"><h1>text</h1></section><hr><section id="projects"><h1>Projects</h1><h2 class="project-tile"><a id="profile-link" target="_blank" href="https://freecodecamp.org">text</a></h2></section><hr></body></html>
+```
+
+```css
+```
```

Comment:
What is this addition here doing?

Reply:
Most solutions for multifile challenges have separate CSS files. This
solution includes a style element in the html to better detect
interactions between the CSSHelper and the header.

Reply:
I am confused, though. This makes it look like there are two `index.html` files, and two `style.css` files for the solution? Which is not possible in the editor.

Reply:
Oh, I see. There are now two solutions, separated by `---`, both with 1 html and 1 css file.

Reply:
Oh. I did not know we had such a thing. Thanks 👍

Reply:
Yeah, it's a thing. We don't use it much and it's not exactly highlighted in the docs, but it exists.